web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java

```diff
@@ -1660,28 +1637,6 @@ public void executing(ChangeSet changeSet, int numChangeSetsToRun) {
 							}
 						}
 						
-						// add demo data only if creating tables fresh and user selected the option add demo data
-						if (wizardModel.createTables && wizardModel.addDemoData) {
-							try {
-								setMessage("Adding demo data");
-								setCompletedPercentage(0);
-								setExecutingTask(WizardTask.ADD_DEMO_DATA);
-								
-								log.debug("executing Liquibase file '{}' ", LIQUIBASE_DEMO_DATA);
-								
-								DatabaseUpdater.executeChangelog(LIQUIBASE_DEMO_DATA,
```

Comment:
Do we still need the LIQUIBASE_DEMO_DATA file?

Reply:
removed!